src/Resources/config/selector.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
-        <service id="sonata.notification.erroneous_messages_selector" class="Sonata\NotificationBundle\Selector\ErroneousMessagesSelector">
+        <service id="sonata.notification.erroneous_messages_selector" class="Sonata\NotificationBundle\Selector\ErroneousMessagesSelector" public="true">
```

Comment:
What about adding a `TODO: remove public in next major` ?

Reply:
I would say: NEXT_MAJOR: Remove public and make use of dependency injection instead of container

Reply:
Or maybe it's the right time to do it ?

On other bundles, we made this change on the stable branch: https://github.com/sonata-project/SonataAdminBundle/pull/5579

Reply:
Well, IMO it should be done, but fixing a bug is fixing a bug... If @core23 wants to invest time on it, good, if not it can be merged as is (I would suggest the next_major comment so another person can work on it)

Reply:
Yes sure.
Either changing the command, either adding a `NEXT_MAJOR` comment to this file.